.github/ISSUE_TEMPLATE/dev_client_installation.yml

```diff
@@ -0,0 +1,56 @@
+name: "\U0001F4F1 Development Client Installation/Setup"
+description: 'Report an issue with setting up or creating a Custom Development Client'
+labels: 'needs review', 'Custom Client - Installation'
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to file a bug report! Please fill out this form as completely as possible.
+  - type: markdown
+    attributes:
+      value: If you leave out sections there is a high likelihood your issue will be closed. If you have a question, not a bug report, please post it on our [forums](https://forums.expo.io/) instead.
+  - type: textarea
+    attributes:
+      label: Summary
+      description: Describe the issue in 1 or 2 sentences
+      placeholder: Clearly describe what the expected behavior is vs. what is actually happening when trying to set up your Development Client. This should be as short as possible, while still communicating all the necessary information. If your summary is just 'X library/method isn't working', then you need to continue debugging yourself and provide more information.
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!
+      options:
+        - managed
+        - bare
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: What platform(s) does this occur on?
+      multiple: true
+      options:
+        - Android
+        - iOS
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Package versions
+      description: 'List the package versions of `expo-dev-client`, `expo-updates`, and any other relevant libraries you are trying to add to your project.'
+  - type: textarea
+    attributes:
+      label: Environment
+      placeholder: Run `expo diagnostics` and paste the output here
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Reproducible demo or steps to reproduce from a blank project
+      description: 'This should include as little code as possible to reproduce the issue. We want to understand the full extent of compatibility issues between external libraries and Custom Development Clients; please help us by simplifying the surface area of the problem as much as you can. Read more guidance [here](https://stackoverflow.com/help/mcve).'
```

Comment:
Could we be more specific about what we're looking for here?  Like being explicit that if your project contains more than 1 non-core module module other than the dev client it is unlikely to be a useful repro case